src/index.html

```diff
@@ -1,13 +1,39 @@
 <!doctype html>
-<html lang="en">
+<html class="page" lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;500;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="product" data-qa="card">
+      <img class="product__img" src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27">
```

Comment:
```suggestion
      <img 
        class="product__img" 
        src="./images/imac.jpeg"
        alt="APPLE A1419 iMac 27"
      >
```
Consider fixing indents according to the code style guide